packages/docusaurus-theme-classic/src/theme/DocBreadcrumbs/index.tsx

```diff
@@ -22,38 +22,49 @@ function BreadcrumbsItemLink({
 }): JSX.Element {
   const className = clsx('breadcrumbs__link', styles.breadcrumbsItemLink);
   return href ? (
-    <Link className={className} href={href}>
-      {children}
+    <Link className={className} href={href} itemProp="item">
+      <span itemProp="name">{children}</span>
     </Link>
   ) : (
-    <span className={className}>{children}</span>
+    <span className={className} itemProp="item name">
+      {children}
+    </span>
   );
 }
 
 // TODO move to design system folder
```

Comment:
I'm not even sure what "design system folder" is, and my intuition tells me I don't like this direction😅 Is that like `theme-common`?

Reply:
@Josh-Cena it's a good practice to have a design system

We already have one anyway: Infima

As we are in React land, we should create a well-encapsulated abstraction on top (think "infima-react") instead of using infima classnames and low-level dom elements everywhere in our theme

This is related to the theme vision in https://github.com/facebook/docusaurus/issues/6649#issuecomment-1042770122